lab-edward/lib/cowsay.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+
+  cowsay: function(greeting) {
+    const cow =
+      '                   \n' +
+      '      ' + greeting + '    \n' +
+      '      ---------------     \n' +
+      '        •           \n' +
+      '         •          \n' +
+      '          •         \n' +
+      '          (___) \n' +
+      '          (o o)_____/ \n' +
+      '            @@\'    \\  \n' +
+      '             \\ ____, / \n' +
+      '            //    // ';
+
+    console.log(cow);
```

Comment:
Your function should return your cow and greeting instead of console logging them